CMakeLists.txt

```diff
@@ -109,9 +111,24 @@ list(APPEND CMAKE_PREFIX_PATH /opt/rocm /opt/rocm/llvm /opt/rocm/hip /opt/rocm/h
 option(ENABLE_HIP_WORKAROUNDS Off)
 set(MIOPEN_INSTALL_CXX_HEADERS Off CACHE BOOL "Install MIOpen's C++ header interface")
 
-option(MIOPEN_USE_COMGR "When possible, use comgr to build kernels instead of offline tools" Off)
+
+# Embedded Build Configuration
+option(MIOPEN_EMBED_BUILD "Build with the set of embed flags." Off)
+option(MIOPEN_USE_COMGR "Use comgr to build kernels instead of offline tools" ${MIOPEN_EMBED_BUILD})
+option(MIOPEN_DISABLE_USERDB "Disable user database access" ${MIOPEN_EMBED_BUILD})
 option(MIOPEN_USE_HIP_KERNELS "Use HIP kernels." On)
 
+# Workaround for comgr 
+if(MIOPEN_USE_COMGR)
+    set(MIOPEN_USE_HIP_KERNELS 0)
```

Comment:
Well, this would exclude building of HIP kernels via COMgr from testing. But this path is working right now (albeit it is sub-optimal in terms of GPU performance), and I would like to see if it breaks.

Can we do like this:
```
# Workaround for comgr 
if(MIOPEN_EMBED_BUILD)
    set(MIOPEN_USE_HIP_KERNELS 0)
endif()
```

Reply:
Fixed

Reply:
@atamazov I've fixed this, please re-review.